compositions/Header/ConnectButton.tsx

```diff
@@ -1,70 +1,72 @@
 import { ConnectButton as RKConnectButton } from '@rainbow-me/rainbowkit'
-import { Button, Flex, Box, Icon } from '@zoralabs/zord'
-import { hideMobile } from 'styles/styles.css'
+import { Button, Flex, Box, Icon, FlexProps } from '@zoralabs/zord'
+import { hideMobile, noTextWrap } from 'styles/styles.css'
 import { EnsAvatar } from '@noun-auction/components/DataRenderers/EnsAvatar'
-import { useWindowWidth } from '@shared'
+import { connectButton } from './Header.css'
 
-export const ConnectButton = () => {
-  const {} = useWindowWidth()
+export interface ConnectButtonProps extends FlexProps {}
 
+export const ConnectButton = ({ ...props }: ConnectButtonProps) => {
   return (
-    <RKConnectButton.Custom>
-      {({ account, chain, openAccountModal, openConnectModal, mounted }) => {
-        return (
-          <div
-            {...(!mounted && {
-              'aria-hidden': true,
-              style: {
-                opacity: 0,
-                pointerEvents: 'none',
-                userSelect: 'none',
-              },
-            })}
-          >
-            {(() => {
-              if (!mounted || !account || !chain) {
+    <Flex {...props} className={[connectButton, 'connect-button-wrapper']}>
+      <RKConnectButton.Custom>
+        {({ account, chain, openAccountModal, openConnectModal, mounted }) => {
+          return (
+            <Box
+              {...(!mounted && {
+                'aria-hidden': true,
+                style: {
+                  opacity: 0,
+                  pointerEvents: 'none',
+                  userSelect: 'none',
+                },
+              })}
+            >
+              {(() => {
+                if (!mounted || !account || !chain) {
+                  return (
+                    <Button
+                      size="sm"
+                      px="x4"
+                      onClick={openConnectModal}
+                      borderRadius="curved"
+                      style={{
+                        whiteSpace: 'nowrap',
```

Comment:
I think there's an atom for whiteSpace

Reply:
Not in this version of zord.
can revisit after upgrading.